.devcontainer/populate_couchdb.sh

```
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Use to create CouchDB system databases and one demo database
+
+# Wait until CouchDB answers
+until curl -fsS "http://localhost:5984/_up" >/dev/null 2>&1; do sleep 2; done
+
+# Variables needed
+now=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Shellcheck (0.10.0)

[warning] now appears unused. Verify use (or export if used externally).

(SC2034)